src/components/dashboard/main-content/course-enrollments/course-cards/tests/BaseCourseCard.test.jsx

```diff
@@ -221,10 +239,10 @@ describe('<BaseCourseCard />', () => {
         endDate={endDate}
         mode="executive-education"
         orgName="some_name"
-        pacing="self"
+        pacing="instructor"
       />,
     );
-    const shouldRenderEndDate = dayjs(startDate) <= dayjs() && type !== 'completed';
+    // const shouldRenderEndDate = dayjs(startDate).isBefore(dayjs(), 'day') && type !== 'completed';
```

Comment:
nit: this line is commented out; is this intentional?

Reply:
Yep,  since this logic is abstracted as a boolean value from the test data from `it.each`. I chose to set true booleans since this set of tests should depend on what the value of type is based on rendering vs rehashing the logic from the component directly in the test.